config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/ToomuchIndium-AAAAAAAAAAAAAAAAAAAMSA==.json

```diff
@@ -29,7 +29,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Milling process is an alternative way to produce Indium, designed for ZPM and UV people. It does not void Aluminium and Galena for no reason, and you can extract Indium from Sphalerite efficiently. In exchange of machine cost and infrastructure needed to process.\n\n§3This multi is LuV, but you have to use in combination with Vacuum Furnace, which is ZPM."
+      "desc:8": "Milling process is an alternative way to produce Indium, designed for ZPM and UV people, although you can start it in LuV. It does not void Aluminium and Galena for no reason, and you can extract Indium from Sphalerite efficiently. In exchange of machine cost and infrastructure needed to process."
```

Comment:
its luv now, not zpm or uv.

Reply:
I figured since the milling line is still balanced around being ZPM/UV I'd keep the text, but if you disagree I'll change it.

Reply:
I am not following, sorry. wdym balanced around zpm/uv? you just downtiered it to luv.

Reply:
The dehydrator multiblock is used for more than just milling. The downtiering mostly makes it so you can use it for rare earth processing in LuV. 

It also means you can do milling in LuV because the other multiblocks in the milling line are not gated to ZPM for some reason. I think making milling LuV is fine, but that doesn't mean that the recipes in the milling line weren't originally intended to be ZPM.

Reply:
this was always the only thing making it zpm

Reply:
I'll update it then, makes more sense probably.